server/models/eventModel.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require('mongoose');
+
+const eventSchema = new mongoose.Schema({
+  name: {
+    required: true,
+    type: String,
+  },
+  date: {
+    required: true,
+    type: Date,
+  },
+  description: {
+    required: true,
+    type: String,
+  },
+  location: {
+    type: {
+      type: String,
+      enum: ['Point'],
+      required: true,
+    },
+    coordinates: {
+      type: [Number],
+      required: true,
+    },
+  },
+  organizerID: {
+    required: true,
+    type: String,
+  },
+  tags: {
+    required: true,
+    type: [String],
+  },
+  registeredUsers: {
+    required: true,
+    type: [String],
+  },
+  activity: {
+    required: true,
+    inventoryDetails: mongoose.Schema.Types.Mixed,
```

Comment:
Remove inventoryDetails (which was from the example) that isn't relevant to the activity.